rmgpy/arkane/main.py

```diff
@@ -225,7 +222,7 @@ def execute(self):
         
         # Initialize the logging system (both to the console and to a file in the
         # output directory)
-        self.initializeLog(self.verbose, os.path.join(self.outputDirectory, 'cantherm.log'))
+        self.initializeLog(self.verbose, os.path.join(self.outputDirectory, 'Arkane.log'))
```

Comment:
I would prefer the file name to be completely lowercase, but it's your choice. The gitignore file should also be updated if you decide to change it.

Reply:
Done. Why should `.gitignore` be changed?
Isn't
```python
# Example output files
examples/*
```
enough?